participants/DBC-AI-Distributed-Cloud-Storage.md

```diff
@@ -0,0 +1,66 @@
+# DBC-AI-Distributed-Cloud-Storage
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name:
+    - *DBC-AI-Distributed-Cloud-Storage*
+
+- Team Slack Handles:
+    - *deepbrainfeng@gmail.com*
+
+## (1) Brief Project Description
+
+*DBC AI Distribute Cloud Storage is a core component of deep brain chain, DeepBrain Chain is a network of distributed artificial intelligence work force, there are a large number of users in the use of GPU, artificial intelligence AI training, we hope to be able to get the user's data can be stored in the GPU machine not only, also can backup stored in filecoin network, so that users can download data at any time to any GPU machine to calculate.*
```

Comment:
Since this is user-provided data - are there any licensing or privacy considerations that need to be considered?

Reply:
Do you have any encryption tools? We can encrypt the data before uploading it, but if we don't have tools for this, we will have to develop this function later

Reply:
i think that's' fine.